adapters/syncswap/src/sdk/config.ts

```diff
@@ -3,5 +3,5 @@ export const enum CHAINS{
 }
 
 export const SUBGRAPH_URLS = {
-    [CHAINS.LINEA]: "https://api.studio.thegraph.com/query/62864/syncswap-graph-linea/v1.4.1.4"
+    [CHAINS.LINEA]: "https://gateway-arbitrum.network.thegraph.com/api/ce0ba3625ebbbd3c4b5a2af394dc8e47/subgraphs/id/3xpZFx5YNWzqemwdtRhyaTXVidKNnjY19XAWoHtvR6Lh"
```

Comment:
@amedrontadora is the usage of arbitrum network intentional here. We need the liquidity on Linea. Please confirm if that will be received from this.

Reply:
This is the address provided by the graph service provider, which can obtain linea data.